core/commands/commit.py

```diff
@@ -71,11 +99,14 @@ def run(self, edit):
             initial_text = COMMIT_HELP_TEXT
 
         if sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_diff"):
-            if option_amend:
+            if self.amending():
                 initial_text += self.git("diff", "HEAD^")
             else:
                 initial_text += self.git("diff", "--cached")
 
+        if sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_summary"):
+            initial_text += self.fetch_commit_summary()
```

Comment:
My next idea was to clean this up so it's just content = " ".[header(), help_text(), footer()].  Since lots of different pieces of the render need to know about amend it seems logical to make it accessible on the class vs passing it into every render method.